src/it/projects/mdep-568-go-offline-exclude/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why all the tildes in the license?

Reply:
removed in https://github.com/apache/maven-dependency-plugin/pull/258. It is a pull request to this branch.

Reply:
I don't understand. I still see them. Do you need to merge to master?